Applications/Spire/Source/Ui/ListItem.cpp

```diff
@@ -18,15 +19,32 @@ ListItem::ListItem(QWidget* component, QWidget* parent)
   m_button->installEventFilter(this);
   layout->addWidget(m_button);
   auto style = get_style(*m_button);
-  style.get(Body()).set(
-    BackgroundColor(QColor::fromRgb(0xFF, 0xFF, 0xFF)));
+  style.get(Body()).
+    set(BackgroundColor(QColor::fromRgb(0xFF, 0xFF, 0xFF))).
+    set(border(scale_width(1), QColor::fromRgb(0, 0, 0, 0))).
+    set(horizontal_padding(scale_width(8)));
   style.get(Hover() / Body()).set(
     BackgroundColor(QColor::fromRgb(0xF2, 0xF2, 0xFF)));
   style.get(Focus() / Body()).set(
-    BackgroundColor(QColor::fromRgb(0x68, 0x4B, 0xC7)));
+    border(scale_width(1), QColor::fromRgb(0x4B, 0x23, 0xA0)));
```

Comment:
This is styling both the size and the color, but the spec only styles the color.